doc/ideas/fr_technical_overview.md

```diff
@@ -33,49 +35,81 @@ Gameplay           | GM8        | villagers need sleep at night, die at work (Ne
+Gameplay           | GM29       | Entirely rebalance milita line. Main gold unit similar to knights/archers. Instead of a counter unit currently.
+Gameplay           | GM30       | Nerf the range on towers early and make them scale much better into the late game.
+Gameplay           | GM31       | Unit queuing doesn't reserve resources. Resources are taken in account, when the unit is actually starting to produce.
+Gameplay           | GM32       | Make denying upgrades more easy, like SC2. often players will destroy buildings to cancel an upgrade in progress and gain the upgrade advantage. maybe in a special mod with less hp for buildings. everybody is free to design such a mod.
+Gameplay           | GM33       | Possibility to put archers/ranged units on a wall/castle to get elevation bonus.
+Gameplay           | GM34       | Quickwalling: first XX% (variable percentage) of a foundation is walkable, setting for e.g. tournaments
+Gameplay           | GM35       | Mesoamerican civilizations (already the most novel civs in the game) may have a unique tech tree. Possibility for a unique tech tree in API.
+Gameplay           | GM36       | Controlled Forest-Burns: Allows the player (technology Researched at the University in the Castle age) to destroy Forests to create more space; this goes much faster/efficient than Chopping, but obviously at the expense of the Lumber and generates Regular Terrain.
+Gameplay           | GM37       | Give seige Rams an upgrade that prevents unit production while it's attacking a building. (5-second debuff that refreshes each attack). Is current API ready to change that easily?
+Gameplay           | GM38       | Bigger maps, map sizes between giant and ludicrous
+Gameplay           | GM39       | Foundation: turns Regular Terrain into Solid Terrain, requires Stone, turns Irregular Terrain into Regular Terrain.
+Gameplay           | GM40       | Different terrain types: 
+Unit AI            | UAI1       | Improved grouping. manually coloring units for teammates. designated unit pings its position periodically. double click on a grouped unit in group formation selects same group not all units of same kind. allow more than 10 groups. more intelligent assigning of units to existing groups. sub-group selections. new unit to group takes same stance like the whole group. multi-group functionality. Better waypoints (AOM-style) - able to queue tasks for units, especially villagers. deselect certain units quickly from a selection by right clicking their portrait.
+Unit AI            | UAI2       | More intelligent units. Emergency evacuation points. Formations and movement when attacking. Dynamic hardlocking on target units when attacking. Aggressive stance for monks (auto-convert). Auto-flee stance. Non-coward stance for villagers. Allow selecting the default stance in production buildings. Set rally point on moving units. Better attack move.
```

Comment:
same here

Reply:
Yep, will melt that together and find some grouping there with the point before (UAI1)

Reply:
done